__tests__/app.test.js

```diff
@@ -84,22 +86,82 @@ describe("core tasks", () => {
         });
     });
     // error handling tests
-    it("GET: responds with 404 and when id not present", () => {
-        return request(app)
-        .get('/api/articles/99999')
+    it("GET 404: when id not in database", () => {
+      return request(app)
+        .get("/api/articles/99999")
         .expect(404)
-        .then(({body}) => {
-            const error = body.message
-            expect(error).toBe('Article not found')
-        })
+        .then(({ body }) => {
```

Comment:
please be careful adding in changes for things that are not in the ticket :) it makes it much more difficult to review your work